distribution/docs/src/main/resources/content/_tables/ContentDirectoryMonitor.adoc

```diff
@@ -40,7 +40,7 @@
 |Processing Mechanism
 |processingMechanism
 |String
-|Choose what happens to the content item after it is ingested. Delete will remove the original file after storing it in the content store. Move will store the item in the content store, and a copy under ./ingested, then remove the original file. (NOTE: this will double the amount of disk space used.) Monitor in place will index the file and serve it from its original location.
+|Choose what happens to the content item after it is ingested. Delete will remove the original file after storing it in the content store. Move will store the item in the content store, and a copy under ./ingested, then remove the original file. (NOTE: this will double the amount of disk space used.) Monitor in place will index the file and serve it from its original location. If in place is used, then the URLResourceReader root resource directories configuration must be updated to allow downloading from the monitored directory (See <<_url_resource_reader, URL Resource Reader>>).
```

Comment:
✏️ Since this is something we're showing to an administrator, we may want to use the configuration name used in the admin console instead of the `URLResourceReader` class name.